src/main/java/fredbot/exception/EmptyDescriptionException.java

```diff
@@ -0,0 +1,4 @@
+package fredbot.exception;
+
+public class EmptyDescriptionException extends Exception{
+}
```

Comment:
I'm not sure why your exception is empty? Do note that there is a code formatting issue as well `Exception{`.